### PR TITLE
Some small improvements to QM docs

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: dimod/quadratic/quadratic_model.py [hunk 1]
```diff
@@ -342,16 +354,31 @@ def shape(self) -> Tuple[int, int]:
...
 
     @forwarding_method
     def add_linear(self, v: Variable, bias: Bias):
-        """Add a quadratic term."""
+        """Add a linear bias to an existing variable."""
         return self.data.add_linear
 
     @forwarding_method
     def add_quadratic(self, u: Variable, v: Variable, bias: Bias):
+        """Add quadratic bias to an interaction between two variables."""
```
Comment (JoelPasvolsky) [inline dimod/quadratic/quadratic_model.py:381]: "Add quadratic bias to an interaction between two variables." should be, "Add quadratic bias as an interaction between two variables" or "Add quadratic bias to a pair of variables."

File: dimod/quadratic/quadratic_model.py [hunk 2]
```diff
@@ -597,6 +629,7 @@ def relabel_variables(self, mapping: Mapping[Variable, Variable],
 
     def relabel_variables_as_integers(self, inplace: bool = True
                                       ) -> Tuple['QuadraticModel', Mapping[Variable, Variable]]:
+        """Relabel the variables as `[0, n)` returning the mapping."""
```
Comment (JoelPasvolsky) [inline dimod/quadratic/quadratic_model.py:632]: "returning the mapping" --> "and return the mapping" if you want the return value described in the one-liner

**Review by JoelPasvolsky — APPROVED** [review]

File: docs/reference/quadratic.rst [hunk 3]
```diff
@@ -0,0 +1,217 @@
+===============================
+Quadratic Models: Unconstrained
+===============================
+
+.. _qm:
```
Comment (JoelPasvolsky) [inline docs/reference/quadratic.rst:5]: For consistency: we usually put this tag at the top of the file

Comment (arcondello) [reply docs/reference/quadratic.rst:5]: I put this one and `_bqm` at the relevant section. Seems more useful that way to me?

Comment (JoelPasvolsky) [reply docs/reference/quadratic.rst:5]: We use these to reference the page; for example, 

image

And the text displayed comes from the subsequent header. We'd want that to be "Quadratic Models: Unconstrained", no?

Comment (arcondello) [reply docs/reference/quadratic.rst:5]: I think we'd still want it to say `Binary Quadratic Models`. I think I am going to leave it for now and you can change it back when you work on it :smile:

**Pull request merged**